Dockerfile

```
@@ -0,0 +1,30 @@
+FROM frolvlad/alpine-miniconda3
+
+LABEL base_image="python:3.8-alpine3.12"
+LABEL about.home="https://github.com/Clinical-Genomics/chanjo"
+LABEL about.documentation="https://www.clinicalgenomics.se/chanjo/"
+LABEL about.tags="bam,NGS,coverage,sambamba,alpine,python3"
+LABEL about.license="MIT License (MIT)"
+
+# Install Sambamba using conda
+RUN conda update -n base -c defaults conda && conda install -c bioconda sambamba
+
+RUN conda install -c conda-forge ruamel.yaml
+
+# Install required libs
+RUN apk update \
+	&& apk --no-cache add bash python3
+
+WORKDIR /home/worker/app
+COPY . /home/worker/app
+
+# Install Chanjo requirements
+RUN pip install -r requirements.txt
+
+# Install the app
+RUN pip install -e .
+
+# Run commands as non-root user
+RUN adduser -D worker
+RUN chown worker:worker -R /home/worker
+USER worker
```

```suggestion
USER worker

ENTRYPOINT = ["chanjo"]
CMD = ["--help"]
```
By doing this one can run the container with 

```docker run --name chanjo```

And then use arguments of course

---

Good suggestion! I'll fix it as you suggest, but without committing exactly your suggestion (you've got an extra = after ENTRYPOINT that breaks the command